### PR TITLE
Implemented UI for rules section

### PR DESCRIPTION
**Pull request by ThaminduChankana**

### CODE REVIEW

**Comment by ThaminduChankana** [comment]
Commit is underway which is implemented being using tailwind

File: src/App.js [hunk 1]
```diff
@@ -23,6 +25,9 @@ function App() {
 						for styling
 					</li>
 				</ul>
+
+				<br></br>
```
Comment (rusiruavb) [inline src/App.js:29]: Use margin bottom instead of `<br />` tag.

File: src/views/rules/Rules.jsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
+import React from 'react'
```
Comment (rusiruavb) [inline src/views/rules/Rules.jsx:1]: Missing semicolon

File: src/views/rules/Rules.jsx [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+                <a href='/'>
+                    <button className='mt-4 w-48 h-10 rounded-md bg-black text-white'>
+                        Get Full Instructions
+                    </button>
+                </a>
+            </div>
+            
+        </div>                    
+    </div>
+  )
```
Comment (rusiruavb) [inline src/views/rules/Rules.jsx:38]: Missing semicolon

**Review by rusiruavb — CHANGES_REQUESTED** [review]
@ThaminduChankana use `yarn` to install the dependencies. `package-lock.json` should not create if you use **yarn**. Remove the package-lock.json and use yarn to install packages.

**Review by rusiruavb — APPROVED** [review]
LGTM

**Pull request merged**